source/AndroidXProject.cshtml

```diff
@@ -188,4 +193,11 @@
   </ItemGroup>
   }
 
+  <ItemGroup>
+    <PackageReference Include="SecurityCodeScan.VS2019" Version="5.6.7">
```

Comment:
From Discord:

> It is for Nuget Auditing only, though I tried to add SAST nuget but it causes some issues I need to check.

If this is true, then let's not add this package.

Reply:
Also note that our pipeline(s) already run the CodeQL static code analysis tool recommended and required by Microsoft security. We should likely continue to rely on their expertise rather than trying to come up with our own solution here.

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=10305045&view=logs&j=784e4eae-0a8d-50ee-7be1-df4337debdeb&t=fbdff2d1-992e-564e-2a8b-113c89c83f2b

Reply:
> If this is true, then let's not add this package.

I pushed to see whether the same problems can be reproduced on CI, but CI built OK. Now I know that updates borked my workloads locally. I had to nuke `dotnet` installations completely and after clean installation and workloads everything was OK.